plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -625,19 +633,32 @@ export class EventsProcessor {
         teamId: number,
         distinctId: string,
         sentAt: DateTime,
-        personUuid: string
-    ): Promise<void> {
+        personUuid: string,
+        properties?: Properties,
+        propertiesOnce?: Properties
+    ): Promise<boolean> {
         const isNewPerson = await this.personManager.isNewPerson(this.db, teamId, distinctId)
         if (isNewPerson) {
             // Catch race condition where in between getting and creating, another request already created this user
             try {
-                await this.db.createPerson(sentAt, {}, teamId, null, false, personUuid.toString(), [distinctId])
+                await this.db.createPerson(
+                    sentAt,
+                    properties || {},
+                    propertiesOnce || {},
+                    teamId,
+                    null,
+                    false,
+                    personUuid.toString(),
+                    [distinctId]
+                )
+                return true
             } catch (error) {
```

Comment:
we should probably make this better and not try to call update later in this case too, but currently this PR doesn't change what happens in case of an error here